test/serializers/attributes_test.rb

```diff
@@ -47,6 +47,16 @@ def test_multiple_calls_with_the_same_attribute
 
         assert_equal([:id, :title, :body], serializer_class._attributes)
       end
+
```

Comment:
I'd rather have a test with multiple attributes and an `if/unless` option since this is the intended use case.